test/unit/helper/labelGenerator_default.js

```diff
@@ -175,14 +175,14 @@ module.exports.tests.default_country = function(test, common) {
     t.end();
   });
 
-  test('country', function(t) {
+  test('country layer labels should only use the `country` field and not the `name`', function(t) {
     var doc = {
-      'name': 'country name',
+      'name': 'source country name',
       'layer': 'country',
       'country_a': 'country code',
-      'country': 'country name'
+      'country': 'hierarchy country name'
     };
-    t.equal(generator(doc),'country name');
+    t.equal(generator(doc),'hierarchy country name');
```

Comment:
I'm kinda concerned that we are taking the hierarchy country, and not the name of the record itself. Not really because of the effect it has, but just because of the precedent it sets (or has already been set and we are now continuing). In any case it may not matter if we decide to get rid of geonames in the near future